test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -397,6 +400,11 @@ public void initClient() throws IOException {
                     if (moduleName.startsWith("serverless-")) {
                         serverless = true;
                     }
+                    if (moduleName.contains("test-multi-project")) {
```

Comment:
Nice! TIL! I tried to do exactly this in my test, but added a method `boolean multiProject()`, this is way nicer. Thanks for adding this!